### PR TITLE
[Dropdown] Add examples of height variation.

### PR DESCRIPTION
**Pull request by exoego**
Adds examples of height variation proposed in https://github.com/fomantic/Fomantic-UI/pull/587

Related: #108 
 
image

### CODE REVIEW

File: server/documents/modules/dropdown.html.eco [hunk 1]
```diff
@@ -1341,6 +1341,21 @@ themes      : ['Default', 'GitHub', 'Material']
       </div>
     </div>
 
+    <div class="dropdown example">
+      <h4 class="ui header">Height</h4>
+      <p>A selection dropdown can be higher (2x of default height) so that more items can be glanced.</p>
+      <select class="ui search higher column dropdown" multiple>
```
Comment (lubber-de) [inline server/documents/modules/dropdown.html.eco]: `column` is not needed here

Comment (exoego) [reply server/documents/modules/dropdown.html.eco]:
```suggestion
      <select class="ui search higher dropdown" multiple>
```

Comment (exoego) [reply server/documents/modules/dropdown.html.eco]: Fixed in https://github.com/fomantic/Fomantic-UI-Docs/pull/109/commits/82adb7844427a1ebf9ac1050bb302d3a39e98383

File: server/documents/modules/dropdown.html.eco [hunk 2]
```diff
@@ -1341,6 +1341,21 @@ themes      : ['Default', 'GitHub', 'Material']
...
+      <h4 class="ui header">Height</h4>
+      <p>A selection dropdown can be higher (2x of default height) so that more items can be glanced.</p>
+      <select class="ui search higher dropdown" multiple>
+        <%- @partial('examples/single/state-options') %>
+      </select>
+    </div>
+
+    <div class="another dropdown example">
+      <p>Highest: 3x of default height.</p>
+      <select class="ui search highest column dropdown" multiple>
```
Comment (exoego) [inline server/documents/modules/dropdown.html.eco]:
```suggestion
      <select class="ui search highest dropdown" multiple>
```

**Pull request merged**